src/test/java/org/apache/maven/plugins/javadoc/AggregatorJavadocReportTest.java

```diff
@@ -173,20 +173,13 @@ private static String readFile( File file )
     {
         String strTmp;

```

Comment:
could pull this into the try block or even the loop initializer

Reply:
@elharo done.